configure.ac

```diff
@@ -138,6 +138,12 @@ AC_ARG_ENABLE([linuxkm-defaults],
     [ENABLED_LINUXKM_DEFAULTS=$ENABLED_LINUXKM]
     )
 
+# FreeBSD Kernel Module
+AC_ARG_ENABLE([bsdkm],
```

Comment:
We probably _do_ want to make the `configure` option `--enable-freebsdkm`, even if internally we are (at least currently) just calling it bsdkm/BSDKM, otherwise we'll be hosed if we need to add OpenBSD later.

Reply:
done, renamed enable-bsdkm -> enable-freebsdkm.